tests/routes/test_validation_api.py

```
@@ -387,8 +389,14 @@ async def mock_get_or_create_collection(*args, **kwargs):
     async def mock_chat_history(*args, **kwargs):
         return "Mock chat history", [], []
 
-    async def mock_get_long_term_facts(*args, **kwargs):
-        return ["Mock fact 1", "Mock fact 2"]
+    async def mock_get_long_term_observations(query: str, embedding_store) -> list[str]:
```

Pylint (3.3.7)

[warning] Unused argument 'query'

(W0613)